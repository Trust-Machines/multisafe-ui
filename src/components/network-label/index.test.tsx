import {render} from '@testing-library/react';

import NetworkLabel from './index';

test('1 Testnet', () => {
    const rendered = render(<NetworkLabel network="testnet"/>);
    expect(rendered.container).toMatchSnapshot()
});

test('2 Mainnet', () => {
    const rendered = render(<NetworkLabel network="mainnet"/>);
    expect(rendered.container).toMatchSnapshot()
});

