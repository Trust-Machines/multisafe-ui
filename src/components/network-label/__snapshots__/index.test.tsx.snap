// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 Testnet 1`] = `
<div>
  <div
    class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorDefault MuiChip-filledDefault css-1fkbu3c-MuiChip-root"
  >
    <span
      class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
    >
      testnet
    </span>
  </div>
</div>
`;

exports[`2 Mainnet 1`] = `
<div>
  <div
    class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorPrimary MuiChip-filledPrimary css-qokti5-MuiChip-root"
  >
    <span
      class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
    >
      mainnet
    </span>
  </div>
</div>
`;
